src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  display: block;
+  justify-content: center;
+  width: 200px;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  background-color: $background-color;
+  box-sizing: border-box;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
+    padding: 0;
+  }
+
+  &__content {
+    margin: 0 16px 16px;
+    padding: 0;
+    width: 166px;
+  }
+
+  &__title {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)